### PR TITLE
Install NVIDIA driver test

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This is currentl a WIP test to see if I can get windows working

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @kkraus14 @mingwandroid (in case either of you have thoughts here)

**Comment by kkraus14** [comment]
I'm not super familiar with the Windows side of things here, especially when using chocolately 😄

**Comment by jakirkham** [comment]
No worries. Do we know anyone within NVIDIA that is?

**Comment by scopatz** [comment]
So whenever Azure comes back online, I think this now works.  The trick is to install cuda using choco in order to get the NvToolsExt part of the package.  

In theory, I think NvToolsExt should be split out into a separate package (nvtx?) but I don't know how much of a headache that would be.

File: .azure-pipelines/azure-pipelines-win.yml [hunk 1]
```diff
@@ -16,6 +16,10 @@ jobs:
     CONDA_BLD_PATH: D:\\bld\\
 
   steps:
+    - script: |
+        choco install cuda --version=11.0.3 -fdv -y --debug
+      displayName: Install Chocolatey Package: cuda --version=11.0.3
```
Comment (jakirkham) [inline .azure-pipelines/azure-pipelines-win.yml:21]: Maybe we need quotes? Seems like Azure is confused about parsing this (guessing the `:` throws it).

```suggestion
      displayName: "Install Chocolatey Package: cuda --version=11.0.3"
```

Comment (scopatz) [reply .azure-pipelines/azure-pipelines-win.yml]: OK, let's try that.  Maybe it needs to be changed in smithy

Comment (scopatz) [reply .azure-pipelines/azure-pipelines-win.yml]: Yeah seems like it does

Comment (jakirkham) [reply .azure-pipelines/azure-pipelines-win.yml]: Thanks Anthony! 😄

**Comment by jakirkham** [comment]
Guessing this PR is no longer WIP? FWIW LGTM.

Thanks for digging into the Windows case! Expect this will be really helpful for maintainers and users 😄

Do you know if Choco also contains NVCC? Not that it is needed here. Just curious about it for users wanting to build packages on Windows.

**Comment by scopatz** [comment]
> Guessing this PR is no longer WIP? FWIW LGTM.

Yep! It is good for me now that it passes!

> Thanks for digging into the Windows case! Expect this will be really helpful for maintainers and users smile

Yeah, of course. I figured, as long as we are going back through issuing packages for old versions, it would be good for Windows to work during that process.

> Do you know if Choco also contains NVCC? Not that it is needed here. Just curious about it for users wanting to build packages on Windows.

Judging by the install size (2.5+ GB), I think that it does. (That is the `choco install cuda` will get you `nvcc`.) However, I didn't specifically check. This would be a good thing to know for sure, but kinda outside of my present interest :wink:

**Comment by jakirkham** [comment]
> Yeah, of course. I figured, as long as we are going back through issuing packages for old versions, it would be good for Windows to work during that process.

Agreed. Thanks!

> > Do you know if Choco also contains NVCC? Not that it is needed here. Just curious about it for users wanting to build packages on Windows.
> 
> Judging by the install size (2.5+ GB), I think that it does. (That is the `choco install cuda` will get you `nvcc`.) However, I didn't specifically check. This would be a good thing to know for sure, but kinda outside of my present interest 😉

Good to know. Thanks for checking.

Ofc. Maybe others will find this of interest and investigate further (cc @jaimergp 😉)

**Comment by leofang** [comment]
@scopatz @jakirkham Question: Does choco also provide the header for nvToolsExt, or just the dll?

**Pull request merged**